Tests/scripts/wait_until_server_ready.py

```diff
@@ -19,7 +19,6 @@
 
 urllib3.disable_warnings()
 
-ARTIFACTS_PATH = os.environ.get('ARTIFACTS_FOLDER')
```

Comment:
Why not just set it to `ARTIFACTS_FOLDER = os.environ.get('ARTIFACTS_FOLDER', './artifacts')` and use that global in the rest of the file?